Source/DafnyCore/DafnyOptions.cs

```diff
@@ -255,6 +255,7 @@ public enum ContractTestingMode {
     public bool FormatCheck = false;
 
     public string CompilerName;
+    public string CompilerExecutable = null;
```

Comment:
Could you remove this field and instead refer to this variable using the expression `options.Get(CommonOptionBag.TargetExecPath)` ?